src/io/fasta.rs

```diff
@@ -13,6 +13,7 @@
 //! let reader = fasta::Reader::new(io::stdin());
 //! ```
 
+use flate2::read::GzDecoder;
```

Comment:
[`MultiGzDecoder`](https://docs.rs/flate2/1.0.13/flate2/bufread/struct.MultiGzDecoder.html) would be a better option here. From the linked docs:
> The multistream format is commonly used in bioinformatics, for example when using the BGZF compressed data.

Reply:
As of niffler 2.1.1 we use `MultiGzDecoder`, relevant PR: https://github.com/luizirber/niffler/pull/28